scene/gui/text_edit.cpp

```diff
@@ -1304,6 +1304,9 @@ void TextEdit::_notification(int p_what) {
 				bottom_limit_y -= theme_cache.style_normal->get_margin(SIDE_BOTTOM);
 			}
 
+			// Draw fuidelines.
```

Comment:
```suggestion
			// Draw guidelines.
```
Although the comment just repeats method name, so it's not really useful.